src/v/net/conn_quota.h

```diff
@@ -13,6 +13,7 @@
 #include "base/seastarx.h"
 #include "base/vassert.h"
 #include "config/property.h"
+#include "rpc/logger.h"
```

Comment:
this is a separate PR the conn_quota is separate from the net::transport which is handled in this PR